lib/classes/PluginManager.js

```diff
@@ -623,7 +625,10 @@ class PluginManager {
 
   assignDefaultOptions(command) {
     _.forEach(command.options, (value, key) => {
-      if (value.default && (!this.cliOptions[key] || this.cliOptions[key] === true)) {
+      if (
+        (value.default || validFalsyDefaultValues.has(value.default)) &&
```

Comment:
I believe right way to go, would be to do `value.default != null` (so ignore just `undefined` and `null`).

Reply:
Ah neat I never thought of trying `==`. That does return `true` for `NaN` though.

Reply:
Yes, and we should not silently ignore `NaN` imo. It looks as configuration error, that we should expose

Reply:
Makes sense. I will push another commit when I get a chance (on the phone atm).

Thanks!